### PR TITLE
[Word] (ChangeTrackingState) Add snippet

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**

### CODE REVIEW

File: playlists-prod/word.yaml [hunk 1]
```diff
@@ -111,6 +111,15 @@
...
+  name: Get content controls by their change tracking state
+  fileName: insert-and-change-tracking-state.yaml
+  description: Inserts content controls then retrieves them by their change tracking state.
+  rawUrl: >-
+    https://raw.githubusercontent.com/OfficeDev/office-js-snippets/prod/samples/word/10-content-controls/insert-and-change-tracking-state.yaml
+  group: Content Controls
+  api_set:
+    WordApi: '1.5'
 - id: word-images-insert-and-get-pictures
   name: Use inline pictures
```
Comment (jipyua) [inline playlists-prod/word.yaml:124]: wrong text about images?

Comment (ElizabethSamuel-MSFT) [reply playlists-prod/word.yaml:124]: @jipyua I'll confirm if that snippet has the correct name

Comment (ElizabethSamuel-MSFT) [reply playlists-prod/word.yaml:124]: Confirmed that the name is correct - the snippet inserts an inline image then retrieves it

File: samples/word/10-content-controls/insert-and-change-tracking-state.yaml [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+            let deletedContentControls = context.document.body
+              .getContentControls()
+              .getByChangeTrackingStates(trackDeletedArray);
+            let normalContentControls = context.document.body.getContentControls().getByChangeTrackingStates(trackNormalArray);
+            addedContentControls.load();
+            deletedContentControls.load();
+            normalContentControls.load();
+            await context.sync();
+            console.log(`Added content controls: ${addedContentControls.items.length}`);
+            console.log(`Deleted content controls: ${deletedContentControls.items.length}`);
```
Comment (AlexJerabek) [inline samples/word/10-content-controls/insert-and-change-tracking-state.yaml:44]: These logs could read like the controls are being added and deleted. Could we make this less ambiguous?

File: samples/word/10-content-controls/insert-and-change-tracking-state.yaml [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+            let trackDeletedArray: Word.ChangeTrackingState[] = [Word.ChangeTrackingState.deleted];
+            let trackNormalArray: Word.ChangeTrackingState[] = [Word.ChangeTrackingState.normal];
+            let paragraphs = context.document.body.paragraphs;
+            paragraphs.load();
+            await context.sync();
+            paragraphs.items[0].insertContentControl("PlainText");
+            paragraphs.items[1].insertContentControl("PlainText");
+            paragraphs.items[2].insertContentControl("RichText");
+            context.document.changeTrackingMode = Word.ChangeTrackingMode.trackAll;
+            paragraphs.items[3].insertContentControl("RichText");
```
Comment (AlexJerabek) [inline samples/word/10-content-controls/insert-and-change-tracking-state.yaml:27]: Is there some significance to these controls being RichText?

**Review by AlexJerabek — CHANGES_REQUESTED** [review]

File: samples/word/10-content-controls/insert-and-change-tracking-state.yaml (hunk 3 above)
Comment (ElizabethSamuel-MSFT) [reply samples/word/10-content-controls/insert-and-change-tracking-state.yaml:27]: Not really. Just showing that they don't only apply to one type of content control.

File: samples/word/10-content-controls/insert-and-change-tracking-state.yaml [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+author: xiruatms
+host: WORD
+api_set:
+    WordApi: '1.5'
+script:
+    content: |
+        $("#insert-controls").on("click", () => tryCatch(insertContentControls));
+        $("#setup").on("click", () => tryCatch(setup));
+
+        async function insertContentControls() {
```
[the remark this answers is not in the document]
Comment (ElizabethSamuel-MSFT) [reply samples/word/10-content-controls/insert-and-change-tracking-state.yaml:14]: I'll see what I can do

File: samples/word/10-content-controls/get-change-tracking-states.yaml [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+            );
+            body.paragraphs
+              .getLast()
+              .insertText(
+                "Use add-in commands to extend the Word UI and launch task panes that run JavaScript that interacts with the content in a Word document. Any code that you can run in a browser can run in a Word add-in. Add-ins that interact with content in a Word document create requests to act on Word objects and synchronize object state.",
+                "Replace"
+              );
+            body.paragraphs.load();
+            await context.sync();
+            body.paragraphs.items[0].insertContentControl("PlainText");
```
Comment (AlexJerabek) [inline samples/word/10-content-controls/get-change-tracking-states.yaml:102]: Can we add a log statement saying that content controls were added to each paragraph? I think that'll make it clearer what the "Delete first content control" button is acting on.

File: samples/word/10-content-controls/get-change-tracking-states.yaml [hunk 6]
```diff
@@ -0,0 +1,174 @@
+order: 9
+id: word-content-controls-get-change-tracking-states
+name: Get change tracking states of content controls
+description: Inserts and deletes content controls then retrieves their change tracking state.
```
Comment (AlexJerabek) [inline samples/word/10-content-controls/get-change-tracking-states.yaml:4]: We already have samples that focus on adding and removing content controls. Should we change this description to focus on the tracking state?

**Review by AlexJerabek — APPROVED** [review]
Thanks for retooling the snippet!! Approving with a couple considerations.

**Pull request merged**